src/pages/AnimeFenix/main.ts

```diff
@@ -0,0 +1,101 @@
+import { pageInterface } from '../pageInterface';
+
+export const AnimeFenix: pageInterface = {
+  name: 'AnimeFenix',
+  domain: 'https://animefenix.tv',
+  languages: ['Spanish'],
+  type: 'anime',
+  isSyncPage(url) {
+    if (url.split('/')[3] === 'ver') {
+      return true;
+    }
+    return false;
+  },
+  sync: {
+    getTitle(url) {
+      const urlParts = url.split('/');
+      if (urlParts.length > 0) {
+        const titleParts = urlParts[urlParts.length - 1].split('-');
+        titleParts.pop();
+        const title = titleParts.join(' ').replace(/\b\w/g, l => l.toUpperCase());
+        return title;
+      }
+      return '';
+    },
+    getIdentifier(url) {
+      return AnimeFenix.sync.getTitle(url);
+    },
+    getOverviewUrl(url) {
+      return j.$('.lista a').first().attr('href') || '';
```

Comment:
Does not seem to work

Reply:
exactly what? I tried it and it did give me the title, identifier and so on.